<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>412</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Terrain Maker</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="separator"/>
    <addaction name="actionFileClose"/>
    <addaction name="separator"/>
    <addaction name="actionFileExport"/>
    <addaction name="separator"/>
    <addaction name="actionFileOpenContainerFolder"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEditDownLevel"/>
    <addaction name="actionEditSendToBack"/>
    <addaction name="actionEditUpLevel"/>
    <addaction name="actionEditBringToFront"/>
    <addaction name="separator"/>
    <addaction name="actionEditSize"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuLanguajes">
     <property name="title">
      <string>&amp;Languajes</string>
     </property>
     <property name="icon">
      <iconset resource="../res/resource.qrc">
       <normaloff>:/images/options-languages.png</normaloff>:/images/options-languages.png</iconset>
     </property>
    </widget>
    <widget class="QMenu" name="menuBars">
     <property name="title">
      <string>Bars</string>
     </property>
     <property name="icon">
      <iconset resource="../res/resource.qrc">
       <normaloff>:/images/options-toolbars.png</normaloff>:/images/options-toolbars.png</iconset>
     </property>
    </widget>
    <addaction name="menuBars"/>
    <addaction name="separator"/>
    <addaction name="menuLanguajes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileNew"/>
   <addaction name="actionFileOpen"/>
   <addaction name="actionFileSave"/>
   <addaction name="separator"/>
   <addaction name="actionFileOpenContainerFolder"/>
   <addaction name="separator"/>
   <addaction name="actionEditSendToBack"/>
   <addaction name="actionEditDownLevel"/>
   <addaction name="actionEditUpLevel"/>
   <addaction name="actionEditBringToFront"/>
   <addaction name="separator"/>
   <addaction name="actionEditSize"/>
  </widget>
  <widget class="QToolBar" name="toolsToolBar">
   <property name="windowTitle">
    <string>Tools bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-exit.png</normaloff>:/images/file-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionFileNew">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-new.png</normaloff>:/images/file-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-open.png</normaloff>:/images/file-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-save.png</normaloff>:/images/file-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/help-about.png</normaloff>:/images/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionEditSendToBack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/edit-send-to-back.png</normaloff>:/images/edit-send-to-back.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Send to back</string>
   </property>
  </action>
  <action name="actionEditBringToFront">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/edit-bring-to-top.png</normaloff>:/images/edit-bring-to-top.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bring to front</string>
   </property>
  </action>
  <action name="actionEditUpLevel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/edit-up-level.png</normaloff>:/images/edit-up-level.png</iconset>
   </property>
   <property name="text">
    <string>Up level</string>
   </property>
  </action>
  <action name="actionEditDownLevel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/edit-down-level.png</normaloff>:/images/edit-down-level.png</iconset>
   </property>
   <property name="text">
    <string>Down level</string>
   </property>
  </action>
  <action name="actionFileExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionFileOpenContainerFolder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-open_container_folder.png</normaloff>:/images/file-open_container_folder.png</iconset>
   </property>
   <property name="text">
    <string>Open container folder</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-close.png</normaloff>:/images/file-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionEditSize">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/edit-size.png</normaloff>:/images/edit-size.png</iconset>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
